pkg/core/managers/apis/dataplane/additional_networking.go

```diff
@@ -19,6 +19,8 @@ func AdditionalInbounds(dataplane *core_mesh.DataplaneResource, mesh *core_mesh.
 	return inbounds, nil
 }
 
+// TODO: is this even used?
```

Comment:
AFAIK this is used to generate an inbound that configures the listener for `PrometheusBackend`. From the code lookup we also use this to match against MeshTrafficPermission MeshFaultInjection etc

Reply:
I assume that at this moment this policies won't take it into account and we need to implement this separately, or extend gathering [listeners](https://github.com/kumahq/kuma/blob/master/pkg/plugins/policies/core/xds/listeners.go) and [clusters](https://github.com/kumahq/kuma/blob/master/pkg/plugins/policies/core/xds/clusters.go) so it extracts Prometheus resources

Reply:
I think we don't need that at the moment, we can add this later on when needed